e2e-test/log/build.gradle

```diff
@@ -186,6 +186,35 @@ task parallelGrokStringSubstituteTest(type: Test) {
     finalizedBy removeDataPrepperNetwork
 }
 
+task opensearchOptionsTest(type: Test) {
```

Comment:
I wonder if we should be adding a new end-to-end test here. There are a lot of different configurations for the OpenSearch sink and it seems best to me to keep most of those tests in the `OpenSearchSinkIT` instead. I'm concerned about this for a couple reasons: 1. Builds take longer, and 2. End-to-end tests can be flaky.

@chenqi0805 , Any thoughts on including a new end-to-end test?